tutorials/CMakeLists.txt

```diff
@@ -20,7 +20,7 @@ add_subdirectory("mc/ising_mc")
 add_subdirectory("mc/simple_mc")
 if (ALPSCore_HAS_MPI)
```

Comment:
If ALPSCore is compiled without MPI, the C++ client code won't compile, because it depends on functions/classes from `alps::mpi`. However, with MPI-free ALPSCore,  there cannot be any interference between ALPSCore and MPI, and the point of the tutorial is moot: one should use MPI as usual, with or without ALPSCore.